hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/VolumeManager.java

```diff
@@ -35,25 +35,6 @@
   void createVolume(OmVolumeArgs args)
```

Comment:
I think it's safe to remove this.  `BenchMarkOMKeyAllocation` does not work anyway:

```
$ bin/ozone genesis -b BenchMarkOMKeyAllocation
...
VOLUME_NOT_FOUND org.apache.hadoop.ozone.om.exceptions.OMException: Volume doesn't exist
	at org.apache.hadoop.ozone.om.BucketManagerImpl.createBucket(BucketManagerImpl.java:130)
	at org.apache.hadoop.ozone.genesis.BenchMarkOMKeyAllocation.setup(BenchMarkOMKeyAllocation.java:86)
	at org.apache.hadoop.ozone.genesis.generated.BenchMarkOMKeyAllocation_keyCreation_jmhTest._jmh_tryInit_f_benchmarkomkeyallocation0_0(BenchMarkOMKeyAllocation_keyCreation_jmhTest.java:338)
	at org.apache.hadoop.ozone.genesis.generated.BenchMarkOMKeyAllocation_keyCreation_jmhTest.keyCreation_Throughput(BenchMarkOMKeyAllocation_keyCreation_jmhTest.java:71)
```